install_sct

```diff
@@ -416,6 +416,12 @@ else
             and verify your internet connection before reinstalling"
     exit $e_status
   fi
+
+  # Install tensorflow for Debian
+  if python -c "import platform; print platform.dist()" | grep -qi 'debian'; then
```

Comment:
is this problem specific to debian or to debian-family, i.e., could some older versions of ubuntu experience the same problem?

Reply:
Really hard to tell without testing on older Ubuntu versions. But since Ubuntu is Debian-based, it can happen. However, it is rarer to see older Ubuntu versions, given that this would raise serious security concerns.

Reply:
Having some Ubuntu containers floating around, I tested this:
```
('Ubuntu', '16.04', 'xenial')
('Ubuntu', '18.04', 'bionic')
```

Now, as for debian, the test could probe for the older version, because eg. debian 8 (jessie) ships with glibc 2.19, and doesn't need to use the old tensorflow:
```
>>> import tensorflow as tf
>>> import platform; print platform.dist()
('debian', '8.10', '')
>>> import sys
>>> sys.executable
'/home/cJ/miniconda2/bin/python'
>>> tf.__version__
'1.5.0'
```

Reply:
It's ok if newer Debian versions use Tensorflow from conda, it will certainly be faster (tensorflow performance on CPU). We need to stick with the TensorFlow 1.3.0 because everyone in the lab is using this version and all our models were validated and developed in this version. There are also breaking API changes in newer TF versions.